src/Traits/HasNameTrait.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace WikibaseSolutions\CypherDSL\Traits;
+
+use LogicException;
+use function bin2hex;
+use function ceil;
+use function openssl_random_pseudo_bytes;
+use function substr;
+
+trait HasNameTrait
+{
+    use ErrorTrait;
+
+    private string $name;
+
+    /**
+     * Returns the length in bytes of the auto-generated part of a name.
+     *
+     * @return int
+     */
+    public static function automaticVariableLength(): int
+    {
+        return 32;
```

Comment:
What is the point of making this a function?

Reply:
I don't remember exactly. I believe I was in a situation at one point where I couldn't define it as a static variable and forgot to change it back.

It's back to being a static variable now :+1: